proto/cosmos/distribution/query.proto

```diff
@@ -5,37 +5,56 @@ import "cosmos/query/pagination.proto";
 import "gogoproto/gogo.proto";
 import "cosmos/cosmos.proto";
 import "cosmos/distribution/distribution.proto";
+import "google/api/annotations.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/distribution/types";
 
 // Query defines the gRPC querier service for distribution module
 service Query {
   // Params queries params of distribution module
-  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {}
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/cosmos/distribution/params";
+  }
 
   // ValidatorOutstandingRewards queries rewards of a validator address
-  rpc ValidatorOutstandingRewards(QueryValidatorOutstandingRewardsRequest) returns (QueryValidatorOutstandingRewardsResponse) {}
+  rpc ValidatorOutstandingRewards(QueryValidatorOutstandingRewardsRequest) returns (QueryValidatorOutstandingRewardsResponse) {
+    option (google.api.http).get = "/cosmos/distribution/{validator_address}/outstanding-rewards";
```

Comment:
I have no idea if grpc-gateway can validate route parameters, but let's say I call `GET /cosmos/distribution/community-pool`, will community-pool be the `validator_address`?

one way to solve that is to have: 
```suggestion
    option (google.api.http).get = "/cosmos/distribution/outstanding-rewards/{validator_address}";
```